wp-content/plugins/wporg-learn/inc/taxonomy.php

```diff
@@ -386,11 +387,11 @@ function register_workshop_series() {
 }
 
 /**
- * Register the Workshop Topic taxonomy.
+ * Register the Topic taxonomy.
  */
-function register_workshop_topic() {
+function register_topic() {
 	$labels = array(
-		'name'                       => _x( 'Topics', 'Topic Plans associated to tutorial.', 'wporg-learn' ),
+		'name'                       => _x( 'Topic', 'Topics associated with the content.', 'wporg-learn' ),
```

Comment:
Typically this is the plural name, is there a reason to switch it to singular? (this is _very_ nitpicky, I know 😅 )

Reply:
I think I thought it was strange for the column to be named as plural, but it's multi select so seems correct 🤷 

Reverted, thanks